test/yarn.lock

```diff
@@ -90,20 +90,8 @@
     to-fast-properties "^2.0.0"
 
 "@callstack/eslint-config@link:..":
-  version "9.1.0"
-  dependencies:
-    babel-eslint "^10.0.3"
-    eslint-config-prettier "^6.7.0"
-    eslint-plugin-flowtype "^4.5.2"
-    eslint-plugin-import "^2.19.1"
-    eslint-plugin-jest "^23.1.1"
-    eslint-plugin-prettier "^3.1.1"
-    eslint-plugin-promise "^4.2.1"
-    eslint-plugin-react "^7.17.0"
-    eslint-plugin-react-hooks "^2.3.0"
-    eslint-plugin-react-native "^3.8.1"
-    eslint-restricted-globals "^0.2.0"
-    prettier "^1.19.1"
+  version "0.0.0"
+  uid ""
```

Comment:
um, why is that?

Reply:
I suspect this is why Travis is failing. Please make sure to pass CI checks :)

Reply:
hmm, when running yarn test this is what got generated, but i think this isn't the problem (generated with latest version of yarn 1.22.4).

it looks like prettier itself it only compatible with node 10+.

i'll remove the node 8 and add node 12.